src/components/ProfileDropdown.vue

```diff
@@ -42,12 +43,21 @@ export default {
     };
   },
   computed: {
+    ...mapGetters(['userState']),
+    getAvatar() {
+      return get(
+          this.userState,
+          'avatar',
+          'https://t4.ftcdn.net/jpg/03/46/93/61/360_F_346936114_RaxE6OQogebgAWTalE1myseY1Hbb5qPM.jpg'
```

Comment:
Imho the default value should be in the avatar component not here.
For example if you pass an undefined/null/empty avatar, inside the avatar component default it to this avatar.
This way we don't have to add this url everywhere we want an avatar